src/logic/calculator.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-else-return */
+/* eslint-disable prefer-template */
```

Comment:
- [ ] Please, consider removing all linter disables from this component and also from the App.js and operate.js components. Let linter makes its direct job - help us to find bugs in the early stages of app building 👍